### PR TITLE
Add user_posts permission

### PR DESCRIPTION
**Pull request by i-am-ains-ly**
Package was throwing an error when requesting the user_posts permission, which is a completely valid permission as listed in the documentation: https://developers.facebook.com/docs/facebook-login/permissions/v2.0#reference-user_posts

### CODE REVIEW

**Comment by logical-and** [comment]
Can you please open pull request at [updated library](https://github.com/logical-and/php-oauth), so I merge your changes.

**Comment by Lusitanian** [comment]
Can't add in that error reporting code, but without that, could merge.

File: src/OAuth/OAuth2/Service/Facebook.php [hunk 1]
```diff
@@ -10,6 +10,7 @@
 use OAuth\Common\Http\Client\ClientInterface;
 use OAuth\Common\Storage\TokenStorageInterface;
 use OAuth\Common\Http\Uri\UriInterface;
+use RC\Modules\DebugModule\DebugComponent;
```
Comment (elliotchance) [inline src/OAuth/OAuth2/Service/Facebook.php]: Remove this.

**Comment by elliotchance** [comment]
Please pull in the latest master to fix the build errors.

**Comment by i-am-ains-ly** [comment]
Have now removed that redundant bit of error reporting code and reverted other small changes to sync back with the master branch. Also added the new publish_pages permission for Facebook. Can I get a merge ?

**Comment by elliotchance** [comment]
There is no diff now. It's just one blank line... Is it possible your changes went in with a previously merged PR?

**Comment by i-am-ains-ly** [comment]
Ahh, correct, those changes have been merged by other contributors!

Thanks anyways dude

**Comment by elliotchance** [comment]
No worries :smile:

**Pull request closed without merging**